jobs/JEVS_AQM_PREP

```diff
@@ -76,21 +75,20 @@ export COMOUT=${COMOUT:-$(compath.py -o ${NET}/${evs_ver})}
 export COMOUTproc=${COMOUTproc:-${COMOUT}/${STEP}/${COMPONENT}/${RUN}.${VDATE}/${MODELNAME}}
 mkdir -p ${COMOUT} ${COMOUTproc}
 
-# alternative choice is dan option different from aqons
+# alternative choice is dan option different from aqobs
```

Comment:
@Ho-ChunHuang-NOAA What is a "dan option"?

Reply:
typo, "an", will fix